bindings/smtp/smtp.go

```diff
@@ -16,6 +16,10 @@ import (
 	"gopkg.in/gomail.v2"
 )
 
+const (
+	defaultPriority = 3
```

Comment:
Should we mark all priority as const?

Reply:
I think it isn't needed.  I added lowest and highest priority as const in order to be more clear.